### PR TITLE
Upgrade to Expressive 2

### LINKED ISSUE #119: Upgrade to expressive 2
https://github.com/zendframework/zend-expressive/releases

https://docs.zendframework.com/zend-expressive/reference/migration/to-v2/

### PR DESCRIPTION
**Pull request by bgaleotti**
Closes #119.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,29 +32,30 @@
...
         "prooph/service-bus" : "^6.0",
         "roave/security-advisories": "dev-master",
         "sandrokeil/interop-config": "^2.0.1",
         "zendframework/zend-config": "^2.6",
-        "zendframework/zend-expressive" : "^1.0.3",
-        "zendframework/zend-expressive-aurarouter": "^1.0",
-        "zendframework/zend-expressive-helpers": "^2.1",
+        "zendframework/zend-expressive" : "^2.0",
+        "zendframework/zend-expressive-aurarouter": "^2.0",
+        "zendframework/zend-expressive-helpers": "^3.0",
```
Comment (prolic) [inline composer.json]: why not 4.0?

File: composer.json [hunk 2]
```diff
@@ -32,29 +32,30 @@
...
         "zendframework/zend-expressive-zendviewrenderer": "^1.1.0",
```
Comment (prolic) [inline composer.json]: 1.4?

File: config/autoload/programmatic-pipeline.global.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<?php
+
```
Comment (prolic) [inline config/autoload/programmatic-pipeline.global.php]: license header missing

File: config/development.config.php.dist [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?php
+
```
Comment (prolic) [inline config/development.config.php.dist]: license header missing

File: config/pipeline.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?php
+
```
Comment (prolic) [inline config/pipeline.php]: license header missing

File: config/routes.php [hunk 6]
```diff
@@ -0,0 +1,94 @@
+<?php
+
```
Comment (prolic) [inline config/routes.php]: license header missing

**Comment by prolic** [comment]
zfcampus/zf-development-mode missing in composer json?

**Comment by bgaleotti** [comment]
also updated `filp/whoops` from 1.1.10 to 2.1.8

**Comment by codeliner** [comment]
thx for the great work @bgaleotti . Now an [excerise](https://github.com/prooph/proophessor-do/issues?q=is%3Aissue+is%3Aopen+label%3Aexercise) ? ;)

**Pull request merged**